### ISSUE TITLE
The example from the dash-table-experiments project does not work. There is an exception.

### ISSUE THREAD

**Issue by alexbeo** [plotly/dash-table-experiments#53]
dash.exceptions.NonExistantPropException: 
Attempting to assign a callback with
the property "filename" but the component
"upload-data" doesn't have "filename" as a property.

Here is a list of the available properties in "upload-data":
['children', 'id', 'contents', 'accept', 'disabled', 'disable_click', 'max_size', 'min_size', 'multiple', 'className', 'className_active', 'className_reject', 'className_disabled', 'style', 'style_active', 'style_reject', 'style_disabled']

**Comment by klnrdknt** [comment]
Yeah, I have the same issue, workign on it currently, not sure what it is caused by. According to the documentation, the upload component should have a 'filename' property. It should work though, if you just erase the part of the filename property in the meantime.

**Comment by klnrdknt** [comment]
Correction: You'd have to erase a ton of functionality from the parser as well.

**Comment by klnrdknt** [comment]
Please make sure you have all the correct versions of dash-core-components installed.

Use:

``pip install dash-core-components==0.21.0rc1

Fixed it for me.
